src/App.tsx

```diff
@@ -1,8 +1,93 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+import { getUsers, getTodos, Todo } from './component/api';
+
+const App = () => {
+  const [sortType, setSortType] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+  const [reversed, setReversed] = useState(false);
+
+  const handleLoadClick = async () => {
+    setLoading(true);
+
+    try {
+      const todosFromServer = await getTodos();
+      const usersFromServer = await getUsers();
+
+      await new Promise(resolve => setTimeout(resolve, 100));
+
+      const todosWithUsers: Todo[] = todosFromServer.map((todo: Todo) => ({
+        ...todo,
+        user: usersFromServer.find(user => user.id === todo.userId),
+      }));
+
+      setTodos(todosWithUsers);
+    } catch (e) {
+      setError('Loading error');
```

Comment:
```suggestion
      setError('Loading error', e);
```
Передавай ошибку тоже

Reply:
<img width="442" alt="зображення" src="https://user-images.githubusercontent.com/38065883/82231139-64b32580-9935-11ea-81b0-a1946a392570.png">
Меняй текст на кнопку на try again